app/css/application.css

```diff
@@ -124,11 +132,6 @@ code {
     @apply leading-code;
 }
 
-[data-react-data] {
-    width: 100%;
-    height: 100%;
-}
```

Comment:
This started playing up. I've removed it for now, and we can do it on an per-component basis as necessary.